### PR TITLE
Modified 4 appliance targets 

### PR DESCRIPTION
**Pull request by prabinovRedhat**
Modified 4 appliance tests to run on real vCenter env:

- appliance_infraprofile_configs
- appliance_localaccounts
- appliance_monitoring
- appliance_networking_dns_hostname

### CODE REVIEW

**Comment by prabinovRedhat** [comment]
cc: @shellymiron @anna-savina

File: tests/integration/targets/vmware_rest_appliance/tasks/appliance_localaccounts.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- name: Assert local accounts are listed
+  ansible.builtin.assert:
+    that:
+      - local_accounts is defined
+      - local_accounts.value is not none
+    fail_msg: "Failed to retrieve the list of local accounts."
+
+- name: Print local accounts
+  ansible.builtin.debug:
+    var: local_accounts
```
Comment (anna-savina) [inline tests/integration/targets/vmware_rest_appliance/tasks/appliance_localaccounts.yml:69]: You didn't restore warn_days. Add always block please to restore warn_days to original value.

File: tests/integration/targets/vmware_rest_appliance/tasks/appliance_monitoring.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      - monitored_items.value | length > 0
+    fail_msg: "Failed to retrieve the list of monitored items."
+
+- name: Print the monitored items
+  ansible.builtin.debug:
+    var: monitored_items
+
+- name: Query the monitoring backend for memory usage
+  vmware.vmware_rest.appliance_monitoring_query:
+    end_time: 2021-04-14T09:34:56.000Z
```
Comment (anna-savina) [inline tests/integration/targets/vmware_rest_appliance/tasks/appliance_monitoring.yml:20]: Maybe try more recent dates. You've got an empty result.

```
    "monitoring_query_result": {
        "changed": false,
        "failed": false,
        "value": [
            {
                "data": [
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    ""
                ],
                "end_time": "2021-04-14T09:34:56.000Z",
                "function": "AVG",
                "interval": "MINUTES5",
                "name": "mem.total",
                "start_time": "2021-04-14T08:34:56.000Z"
            }
```

File: tests/integration/targets/vmware_rest_appliance/tasks/appliance_localaccounts.yml (hunk 1 above)
Comment (prabinovRedhat) [reply tests/integration/targets/vmware_rest_appliance/tasks/appliance_localaccounts.yml:68]: fixed

File: tests/integration/targets/vmware_rest_appliance/tasks/appliance_monitoring.yml (hunk 2 above)
Comment (prabinovRedhat) [reply tests/integration/targets/vmware_rest_appliance/tasks/appliance_monitoring.yml]: fixed

**Review by anna-savina — APPROVED** [review]
LGTM

**Review by shellymiron — APPROVED** [review]

**Pull request merged**